.github/workflows/api-ci.yaml

```diff
@@ -0,0 +1,31 @@
+name: harmony-api
+on: pull_request
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: ./harmony-api
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: "18"
```

Comment:
Version 18 and alpine 18 linux in docker could have differences, but it works fine. So accepting and leaving this comment for the record